examples/demo/src/invoices/InvoiceList.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import {
+    List,
+    Datagrid,
+    TextField,
+    DateField,
+    ReferenceField,
+    NumberField,
+    ShowGuesser,
+    Filter,
+    DateInput,
+} from 'react-admin';
+
+import FullNameField from '../visitors/FullNameField';
+import AddressField from '../visitors/AddressField';
+
+const ListFilters = props => (
+    <Filter {...props}>
+        <DateInput source="date_gte" alwaysOn />
+        <DateInput source="date_lte" alwaysOn />
+    </Filter>
+);
+
+const InvoiceList = props => (
+    <List {...props} filters={<ListFilters />} perPage={25}>
+        <Datagrid expand={<ShowGuesser />}>
```

Comment:
Using the ShowGuesser does not allow you to override the `title` prop. It can be seen in the last screenshot: there are two titles concatenated (list and show) when expanding an line

Reply:
Fixed in the next commit. I only used the guesser temporarily.